python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -619,7 +619,11 @@ def assembleMetadata(self, coaddExposure, tempExpRefList, weightList):
             self.inputRecorder.addVisitToCoadd(coaddInputs, tempExp, weight)
         coaddInputs.visits.sort()
         if self.config.doPsfMatch:
-            psf = self.config.modelPsf.apply()
+            # set PSF to modelPsf with maximum dimensions
+            modelPsfList = [tempExp.getPsf() for tempExp in tempExpList]
+            modelPsfDimList = [modelPsf.computeBBox().getDimensions() for modelPsf in modelPsfList]
+            maxDim = max(modelPsfDimList)
```

Comment:
I'm surprised `max(dimList)` works.  I can't figure out why it does, but I'm not familiar with pybind and maybe I missed something.